python/dgl/graphbolt/impl/ondisk_dataset.py

```diff
@@ -62,13 +67,19 @@ def preprocess_ondisk_dataset(
         )
 
     # 0. Check if the dataset is already preprocessed.
-    preprocess_metadata_path = os.path.join("preprocessed", "metadata.yaml")
+    processed_dir_prefix = "preprocessed"
+    preprocess_metadata_path = os.path.join(
+        processed_dir_prefix, "metadata.yaml"
+    )
     if os.path.exists(os.path.join(dataset_dir, preprocess_metadata_path)):
-        print("The dataset is already preprocessed.")
-        return os.path.join(dataset_dir, preprocess_metadata_path)
+        if force_reload:
+            shutil.rmtree(os.path.join(dataset_dir, processed_dir_prefix))
+            print("The preprocessed dataset has been removed.")
```

Comment:
the log is a bit confusing. just notice user that we're re-preprocessing, so we removed previously preprocessed one or existing one.

Reply:
Will this log be better?
```
The on-disk dataset is re-preprocessing, so the existing preprocessed dataset has been removed.
```